changelog/_unreleased/2021-04-01-resolve-symfony-5-1-deprecations.md

```diff
@@ -0,0 +1,6 @@
+---
+title: Resolve Symfony 5.1 deprecations
+issue: NEXT-14586
```

Comment:
Please complete the meta information according to [our contribution guideline](https://developer.shopware.com/docs/resources/guidelines/code/contribution#requirements-for-a-successful-pull-request) (`author`, `author_email`).

Reply:
I have added `author`, `author_email` and `author_github`.